### PR TITLE
Bulk liq orders

### PR DESCRIPTION
**Pull request by bthaile**
wire up bulk order creation. 

https://github.com/augurproject/augur/issues/3285

image

image

### CODE REVIEW

File: packages/augur-ui/src/modules/modal/modal.styles.less [hunk 1]
```diff
@@ -382,14 +382,20 @@ div.ButtonsRow:last-of-type {
       margin-top: 1rem;
     }
   }
+
+  > span:last-of-type {
+    display: flex;
+    flex-flow: row-reverse;
+    padding: 0 1.5rem 0.25rem;
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/modal/modal.styles.less]: should use constants

**Review by phoebemirman — APPROVED** [review]

File: packages/augur-ui/src/modules/modal/modal.styles.less (hunk 1 above)
Comment (bthaile) [reply packages/augur-ui/src/modules/modal/modal.styles.less]: will do

**Comment by bthaile** [comment]
updated to use size constants.

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 💜

 This link will always point to this PR's latest changes: https://peek.run/r9zhxjz

**Pull request merged**